app/src/main/java/com/sunasterisk/weather/screen/weather/WeatherContract.kt

```diff
@@ -0,0 +1,17 @@
+package com.sunasterisk.weather.screen.weather
+
+import java.lang.Exception
+import com.sunasterisk.weather.data.model.Weather
+import com.sunasterisk.weather.utils.BasePresenter
+
+interface WeatherContract {
+
+    interface View {
+        fun currentWeather(weather: Weather)
+        fun onError(exception: Exception)
```

Comment:
currentWhether là gì ?? 
View sẽ chứa những hành vi mang ý nghĩa hiển thị, thông báo, ... 
nó không cần biết logic của nó là gì vì logic sẽ được xử lý trong presenter 
nên tên phương thức cũng phải đặt hợp lý 
```suggestion
        fun showCurrentWeather(weather: Weather)
        fun showError(exception: Exception)
```